src/libraries/System.Runtime/tests/System/Type/TypeTests.cs

```diff
@@ -528,6 +528,7 @@ public void GetTypeByName_Invalid(string typeName, Type expectedException, bool
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/52393", typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser), nameof(PlatformDetection.IsMonoAOT))]
```

Comment:
is this going to fail on any mono AOT?

Reply:
@vargaz ^

Reply:
It might, Type.GetType () might need a stack walk to find the 'current' assembly.